package.json

```diff
@@ -24,13 +24,18 @@
     },
     "devDependencies": {
         "@types/bluebird": "^3.5.33",
+        "@types/mocha": "^10.0.0",
         "@types/node": "^12",
         "@types/request": "^2.48.8",
+        "chai": "^4.3.7",
```

Comment:
We should use `jest` instead of `chai`, per [Whispir's Greenfield Architecture guide](https://whispir.atlassian.net/wiki/spaces/Whispir/pages/758874153/Whispir+Greenfield+Architecture+Reference#JavaScript-Specific-Technologies).